content/src/display-content/roadmap-sidebar-cards/go-to-profile-nudge.md

```diff
@@ -1,10 +1,11 @@
 ---
 id: go-to-profile
-ctaText: "Go to Profile"
-borderColor: primary-light
+header: Customize Your Experience
 hasCloseButton: false
+ctaText: ""
```

Comment:
pebble, should we delete this?

Reply:
It looks like this is a common pattern we have. Other cards `roadmap-sidebar-cards` have an empty string for this value. I'm not positive, but we may need to have something here unless we were to restructure the `config.yml` file. `registered-for-taxes` does use this so it may just need to stay.

Reply:
yea i just followed a similiar pattern